### PR TITLE
Removed redudant [SE] block

### PR DESCRIPTION
**Pull request by vinnie1234**
Not sure if the [SE] block in the reg was added on purpose, it may cause an error in my setup where the vehicle is not found while setting up the PSA Car Controller.

### CODE REVIEW

**Comment by vinnie1234** [comment]
VR3UPHNKSLT******

Delete the last section for privacy! 208 GT, buildyear 2020...

File: psa_car_controller/psacc/resources/car_models.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@
   battery_power: 0
   fuel_capacity: 44
   abrp_name:
-  reg: VR3UPHN[SE].*
+  reg: VR3UPHN.*
```
Comment (flobz) [inline psa_car_controller/psacc/resources/car_models.yml:100]:
```suggestion
  reg: VR3UPHN[SEK].*
```

**Comment by flobz** [comment]
please apply my sugestion... "[SEK]" means S or E or K

**Comment by vinnie1234** [comment]
That's works fine, tnx!

**Pull request merged**